### DISCUSSION TITLE
Adding more integrations

### DISCUSSION THREAD

**Discussion by geistchevalier** [Security-Onion-Solutions/securityonion#12058] [2.4] [answered]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

6

### RAM

64GB

### Storage for /

500GB

### Storage for /nsm

2TB

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I noticed that there are some integrations for Elastic Fleet that are not available in the current security onion version (https://docs.securityonion.net/en/2.4/elastic-fleet.html#integrations), are there any plans to add integrations like mssql, mysql, iis, nginx, docker, etc to SO?
or is there a way for me to manually add the integrations that I need to SO?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by ghost** [comment]
I'm curious about the same thing. I tried searching for integration to collect Kubernetes API logs, but couldn't find a integrations.
[elastic-package integration registry](https://discuss.elastic.co/t/how-to-add-custom-integration-into-elastic-agent-integration-package-registry/326740) 
I found similar content on this topic here, but I'm trying to find a solution or any methods to resolve it, and I haven't been successful so far.

**Comment by dougburks** [comment]
We plan to add integrations to the supported list at https://docs.securityonion.net/en/2.4/elastic-fleet.html#integrations as time goes on. Here's what we're adding for the upcoming 2.4.40 release: https://github.com/Security-Onion-Solutions/securityonion/issues/11958.

**Reply by geistchevalier** [reply]
Can I assume that the integrations mentioned in this discussion will possibly be included in future updates? or do I need to create a request/issue to have them added in the future?

**Reply by dougburks** [reply, marked as answer]
We will take a look at additional integrations as time allows. Preference is given to integration requests from paid customers.
